apps/settings/js/panels/screen_lock/panel.js

```diff
@@ -9,7 +9,13 @@ define(function(require) {
 
     return SettingsPanel({
       onInit: function(panel) {
-        screenLock.onInit(panel);
+        var elements = {
+          panel: panel,
+          lockscreenEnable: panel.querySelector('.lockscreen-enable'),
+          passcodeEnable: panel.querySelector('.passcode-enable'),
+          passcodeEditButton: panel.querySelector('.passcode-edit')
```

Comment:
Yes, it's a convention between panels to isolate all real query in `panel.js`, make script (screen_lock.js) independent from real html and make it easier to test.

FYR
https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/js/panels/developer/panel.js